jhack/conf/conf.py

```diff
@@ -85,4 +92,81 @@ def print_current_config():
     CONFIG.pprint()
 
 
+class Denied:
```

Comment:
Something to consider: perhaps include docstrings 🙂 . You might be interested in a linting tool: https://github.com/jdkandersson/flake8-docstrings-complete

Reply:
I was focusing on the behaviour for now, but will do that